irohad/ordering/impl/on_demand_connection_manager.cpp

```diff
@@ -25,13 +25,33 @@ OnDemandConnectionManager::OnDemandConnectionManager(
   initializeConnections(initial_peers);
 }
 
-void OnDemandConnectionManager::onTransactions(CollectionType transactions) {
+void OnDemandConnectionManager::onTransactions(transport::RoundType round,
```

Comment:
Maybe rework it with const ref instead?

Reply:
This will require changes to ordering service and other components, which implement notification interface. It was probably not the best idea to reuse the same interface in both client and server side, but I would rather discuss it in a separate task.